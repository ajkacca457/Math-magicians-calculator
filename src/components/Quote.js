import React from 'react';

const Quote = () => (
  <div className="quote">
    <h3 style={{ width: '60%', margin: ' 15% auto', fontSize: '150%' }}>
      Mathematics is not about numbers, equations, computations,
      or algorithms: it is about understanding.
      {' '}
      <br />
      <span style={{ color: 'orange' }}>â€“William Paul Thurston</span>
    </h3>
  </div>
);

export default Quote;
