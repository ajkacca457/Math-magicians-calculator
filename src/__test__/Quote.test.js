import React from 'react';
import TestRenderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Quote from '../components/Quote';

describe('Quote Component', () => {
  it('renders correctly', () => {
    const quotetree = TestRenderer
      .create(<Quote />)
      .toJSON();
    expect(quotetree).toMatchSnapshot();
  });
});
