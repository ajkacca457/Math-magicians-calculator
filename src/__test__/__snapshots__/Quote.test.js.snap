// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Component renders correctly 1`] = `
<div
  className="quote"
>
  <h3
    style={
      Object {
        "fontSize": "150%",
        "margin": " 15% auto",
        "width": "60%",
      }
    }
  >
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.
     
    <br />
    <span
      style={
        Object {
          "color": "orange",
        }
      }
    >
      â€“William Paul Thurston
    </span>
  </h3>
</div>
`;
